src/main.js

```diff
@@ -64,7 +64,7 @@ function run() {
     }
 
     startTime = new Date()
-    output = execSync(command, {timeout, env, stdio: 'inherit'}).toString()
+    output = execSync(command, {timeout, env, stdio: 'inherit'})?.toString()
```

Comment:
Is this like `&.` in ruby where it will only continue down the chain if there's a return value of the first part? I did some googling and it looks like that's what's happening here. If so, that's super cool. I didn't realize JS had this operator.

Reply:
Yes it is!